packages/apollo-server-types/src/index.ts

```diff
@@ -265,12 +270,14 @@ export interface CachePolicy extends CacheHint {
    * make the policy more restrictive or less restrictive.
    */
   replace(hint: CacheHint): void;
+
```

Comment:
Curious where these extra newlines in some files came from, given that we do have prettier enforcement now...

Reply:
I'm curious as well, my IntelliJ adds those newlines, but it looks CI prettier enforcement doesn't mind?

Reply:
Ah, interesting. Ideally it's nice not to put in unnecessary style changes (would be better to configure IntelliJ to use prettier instead of its own formatting) but not gonna sweat it for this PR.

Reply:
What's odd is that I have IntelliJ configured to use prettier on format, but it's still adding the newlines (maybe it's a prettier version mismatch?)
<img width="996" alt="Screen Shot 2021-07-22 at 2 32 45 PM" src="https://user-images.githubusercontent.com/2695578/126690912-0abc69cd-2521-4c44-af89-7f2fef45fab6.png">

But yeah, I'm down to worry about that later.